File: android-wear-os-java/app/src/main/java/io/github/aidenkoog/android_wear_os_java/MainActivity.java
package io.github.aidenkoog.android_wear_os_java;

import android.app.Activity;
import android.os.Bundle;
import android.widget.TextView;

import io.github.aidenkoog.android_wear_os_java.databinding.ActivityMainBinding;

public class MainActivity extends Activity {

    private TextView mTextView;
    private ActivityMainBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        binding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());

        mTextView = binding.text;
    }
}

File: android-wear-os-kotlin/app/src/main/java/io/github/aidenkoog/android_wear_os_kotlin/MainActivity.kt
package io.github.aidenkoog.android_wear_os_kotlin

import android.app.Activity
import android.os.Bundle
import io.github.aidenkoog.android_wear_os_kotlin.databinding.ActivityMainBinding

class MainActivity : Activity() {

    private lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

    }
}